packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -253,6 +282,30 @@ const ContractDetails = ({
                     label={localize('Start time')}
                     value={toGMTFormat(epochToMoment(Number(date_start))) || ' - '}
                 />
+                {isLookBacksContract(contract_type) && (
+                    <React.Fragment>
+                        {contract_type === CONTRACT_TYPES.LB_HIGH_LOW ? (
```

Comment:
There are 3 types of LockBacks contracts: `LB_HIGH_LOW` , `LB_PUT`  and `LB_CALL`. For the first one we should show 2 fields in contract audit: one for Hight indicative spot, second- for Low. Regarding `LB_PUT` or `LB_CALL` - we should show only one corresponding field (Hight indicative spot for `LB_PUT` and Low indicative spot for `LB_CALL`)